index.html

```diff
@@ -314,6 +315,34 @@ <h6>CSS</h6>
       </div>
     </section>
 
+    <section class="contact-me">
+      <div class="contact-form">
+        <h2> Contact me</h2>
+      </div>
+      <p>If you have an application you are interested in developing, a feature that you need built or a project that
+        needs coding. I'd love to help with it.</p>
+
+
+      <div class="form-section">
+        <form action="https://formspree.io/f/xrgdqvko" method="post">
+          <ul class="form">
+            <li class="form-name">
+              <input type="text" name="name" id="name" placeholder="Full name" maxlength="30" required />
+            </li>
+            <li>
+              <input type="email" name="email" id="email" placeholder="Email address" required />
+            </li>
+            <li>
+              <textarea type="textarea" name="textarea" id="textarea" placeholder="Enter text here" maxlength="500"
+                required></textarea>
+            </li>
+          </ul>
+          <button class="button-contact" type="submit">Get in Touch</button>
+        </form>
+      </div>
+
+    </section>
```

Comment:
- [ ] Your form is working fine, but on some small mobile screens, the form section is not well-formatted and is not centered on the page like in the Figma template and the form itself is close to the left border of the page. Please try to center the form section, `align-items` is a good way to center the elements of a section, so your portfolio can have a good UI on all screen sizes.
 -Your design:
<img width="468" alt="image" src="https://user-images.githubusercontent.com/61919617/193848392-22140566-b3fe-48a0-bdce-58ef12d4b7ee.png">

-Figma template:
<img width="316" alt="image" src="https://user-images.githubusercontent.com/61919617/193848460-d5df3d3f-6926-4f2d-b921-fa504c594fa6.png">